tests/test_eload_validation.py

```diff
@@ -23,9 +24,11 @@ def setUp(self) -> None:
         self.sv_validation = EloadValidation(70)
         # Used to restore test config after each test
         self.original_cfg = deepcopy(self.validation.eload_cfg.content)
+        self.original_cfg = deepcopy(self.sv_validation.eload_cfg.content)
```

Comment:
If you use a different variable for this, you shouldn't have to modify ELOAD_2's config and all your tests should pass :+1: